Dockerfile

```diff
@@ -0,0 +1,156 @@
+# Base image
+FROM postgres:bullseye
+
+# Set environment variables
+ARG USER_NAME=fileserver
+ENV POSTGRES_USER=$USER_NAME
+ENV POSTGRES_PASSWORD=changeme
+ENV POSTGRES_DB=sessionfiles
+ENV PYTHONPATH=/usr/lib/python3.9:/usr/lib/python3:/usr/local/lib/python3.9/site-packages:$PYTHONPATH
+
+# Copy the database schema across (intentionally rename to 'schema.sql' in order for it to be executed)
+COPY schema.pgsql /docker-entrypoint-initdb.d/schema.sql
+
+# Install required dependencies
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    python3 \
+    python3-flask \
+    python3-coloredlogs \
+    python3-pycryptodome \
+    python3-nacl \
+    python3-requests \
+    python3-pip \
+    uwsgi \
+    uwsgi-emperor \
+    uwsgi-plugin-python3 \
+    nginx \
+    libpq-dev \
+    build-essential \
+    curl \
+    lsb-release \
+    supervisor
+
+# Install the Oxen apt repository public signing key and add the Oxen apt repository
+RUN curl -so /etc/apt/trusted.gpg.d/oxen.gpg https://deb.oxen.io/pub.gpg && \
+    echo "deb https://deb.oxen.io $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/oxen.list && \
+    apt-get update && apt-get install -y \
+    python3-pyonionreq \
```

Comment:
See other comment about swapping this to python3-session-util

Reply:
This has been updated now